backend/api_corridor_test.go

```diff
@@ -13,43 +13,45 @@ import (
 )
 
 var update = flag.Bool("update", false, "update the golden files of this test")
-var NGNTtoEURTparams = "/?sourceCode=NGNT&sourceIssuer=GAWODAROMJ33V5YDFY3NPYTHVYQG7MJXVJ2ND3AOGIHYRWINES6ACCPD&destCode=EURT&destIssuer=GAP5LETOV6YIE62YAM56STDANPRDO7ZFDBGSNHJQIYGGKSMOZAHOOS2S"
+var NGNTtoEURTCorridor = "/corridor?sourceCode=NGNT&sourceIssuer=GAWODAROMJ33V5YDFY3NPYTHVYQG7MJXVJ2ND3AOGIHYRWINES6ACCPD&destCode=EURT&destIssuer=GAP5LETOV6YIE62YAM56STDANPRDO7ZFDBGSNHJQIYGGKSMOZAHOOS2S"
```

Comment:
I don't think we need to export this variable - preference to keep unexported unless definitely needed elsewhere.